### ISSUE TITLE
Config and Dependency documentation

### ISSUE THREAD

**Issue by lishanl** [SatelCreative/spylib-legacy#6]
1. Document any configuration needed. e.g.  [Oauth](https://github.com/SatelCreative/spylib/blob/main/spylib/shopify/oauth/config.py) 
2. Document any dependencies needed to be added to requirements.txt e.g.  `shortuuid` used in [Utils](https://github.com/SatelCreative/spylib/blob/main/spylib/utils/misc.py). may also put it in the python base image

**Comment by lishanl** [comment]
`shortuuid` is in the python base image but azure function is not running in docker, that's when it's missing the necessary dependencies. I think it's good to document the dependencies required by spylib 😅

**Comment by hillairet** [comment]
Yeah I need to figure out how to easily make the list of repos needed.
